test/services/groups/destroy_service_test.rb

```diff
@@ -36,5 +36,27 @@ def setup
         Groups::DestroyService.new(@group, @user).execute
       end
     end
+
+    test 'metadata summary updated after group deletion' do
+      # Reference group/projects descendants tree:
+      # group12 < subgroup12b (project30 > sample 33)
+      #    |
+      #    ---- < subgroup12a (project29 > sample 32) < subgroup12aa (project31 > sample34 + 35)
+      @group12 = groups(:group_twelve)
+      @subgroup12a = groups(:subgroup_twelve_a)
+      @subgroup12b = groups(:subgroup_twelve_b)
+      @subgroup12aa = groups(:subgroup_twelve_a_a)
+
```

Comment:
Can you add assert statements for what the metadata summary is equal to before calling the destroy service as well

Reply:
Updated in f76e47740b96955688a5b0d9e0c1c418deacaad4